src/accelerate/data_loader.py

```diff
@@ -17,6 +17,7 @@
 
 import torch
 from torch.utils.data import BatchSampler, DataLoader, IterableDataset
+from torch.utils.data.datapipes.iter.combinatorics import ShufflerIterDataPipe
```

Comment:
This will break compatibility with older versions of PyTorch as I believe datapipes were just introduced in PyTorch 1.11.